all-repos.yaml

```diff
@@ -247,3 +247,4 @@
 - https://github.com/hhatto/autopep8
 - https://github.com/rhysd/actionlint
 - https://github.com/hija/clean-dotenv
+- https://github.com/dbhagen/blocking-comment
```

Comment:
this can be entirely replaced with a no code `pygrep` pattern

Reply:
I don't disagree, that's how I started, but having a specific hook that displays a specific message on error was more applicable to my needs. If that doesn't fit the community contributed hooks, that's cool, just something I use this specific way and figured others might be interested in.

Reply:
Feel free to merge in or close the PR, either way.